src/contact.ts

```diff
@@ -223,6 +302,13 @@ export class Contact implements Sayable {
 
   /**
    * find contact by `name` or `alias`
+   *
+   * @static
+   * @param {ContactQueryFilter} [queryArg]
+   * @returns {Promise<Contact[]>}
+   * @description If use Contact.findAll() get the contact list of the bot.
```

Comment:
Put @description at the beginning of the comment block.

Because the comments at the beginning of the comment block is default to be @description.